cmd/macaroon/webtest-pj.herokuapp.com.yaml

```diff
@@ -0,0 +1,10 @@
+tokens:
+  - id: "webtest-pj.herokuapp.com brave-tshirt v1"
+    version: 2
+    location: "webtest-pj.herokuapp.com"
+    first_party_caveats:
+      - sku: "user-wallet-vote"
```

Comment:
should we change this sku to something else since presumably this isn't a vote we're purchasing?